http/handler_test.go

```diff
@@ -180,6 +181,7 @@ func TestSysMounts_headerAuth(t *testing.T) {
 				},
 				"local":     false,
 				"seal_wrap": false,
+				"options":   interface{}(nil),
```

Comment:
Nit: In here and elsewhere I _think_ you can simply do `"options": nil`

Reply:
Yes, unless the intent was to do `map[string]interface{}(nil)`